Sources/FluentKit/Properties/Child.swift

```diff
@@ -0,0 +1,241 @@
+extension Model {
+    public typealias Child<To> = ChildProperty<Self, To>
+        where To: FluentKit.Model
+}
+
+// MARK: Type
+
+@propertyWrapper
+public final class ChildProperty<From, To>
+    where From: Model, To: Model
+{
+    public enum Key {
+        case required(KeyPath<To, To.Parent<From>>)
+        case optional(KeyPath<To, To.OptionalParent<From>>)
+    }
+
+    public let parentKey: Key
+    var idValue: From.IDValue?
+
+    public var value: To?
+
+    public init(for parent: KeyPath<To, To.Parent<From>>) {
+        self.parentKey = .required(parent)
+    }
+
+    public init(for optionalParent: KeyPath<To, To.OptionalParent<From>>) {
+        self.parentKey = .optional(optionalParent)
+    }
+
+    public var wrappedValue: To {
+        get {
+            guard let value = self.value else {
+                fatalError("Child relation not eager loaded, use $ prefix to access: \(name)")
+            }
+            return value
+        }
+        set {
+            fatalError("Child relation is get-only.")
+        }
+    }
+
+    public var projectedValue: ChildProperty<From, To> {
+        return self
+    }
+    
+    public var fromId: From.IDValue? {
+        return self.idValue
+    }
+
+    public func query(on database: Database) -> QueryBuilder<To> {
+        guard let id = self.idValue else {
+            fatalError("Cannot query child relation from unsaved model.")
+        }
+        let builder = To.query(on: database)
+        switch self.parentKey {
+        case .optional(let optional):
+            builder.filter(optional.appending(path: \.$id) == id)
+        case .required(let required):
+            builder.filter(required.appending(path: \.$id) == id)
+        }
+        return builder
+    }
+
+    public func create(_ to: [To], on database: Database) -> EventLoopFuture<Void> {
+        guard let id = self.idValue else {
+            fatalError("Cannot save child to unsaved model.")
+        }
+        to.forEach {
+            switch self.parentKey {
+            case .required(let keyPath):
+                $0[keyPath: keyPath].id = id
+            case .optional(let keyPath):
+                $0[keyPath: keyPath].id = id
+            }
+        }
+        return to.create(on: database)
+    }
```

Comment:
This method kind of goes against the idea of there being a single child.

Reply:
Thanks for reviewing the code. You are absolutely right in both points.
- The method is a leftover I missed. I already pushed new commit to PR.
- Current implementation requires `.unique` database constraint to ensure `Child` uniqueness. I'm open to any ideas if there is a feasible way to check whether `Child` exists during creation.